sonic_y_cable/y_cable_base.py

```diff
@@ -133,10 +136,10 @@ def check_active_linked_tor_side(self, port):
                 an Integer, the actual physical port connected to a Y cable
 
         Returns:
-            an Integer, 1 if TOR A is actively linked and routing(TOR 1).
-                      , 2 if TOR B is actively linked and routing(TOR 2).
-                      , 0 if nothing linked and actively routing
-                      , -1 if checking which side linked for routing API fails.
+            One of the following predefined constants:
+                TARGET_TORA, if TOR A is actively linked and routing(TOR A).
+                TARGET_TORB, if TOR B is actively linked and routing(TOR B).
```

Comment:
Seems like the parenthesized text is redundant now. Suggest removing.

Reply:
fixed